### ISSUE TITLE
Istioctl should display data plane version/version skew

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#17172]
Right now it is very easy to have version skew between control plane and data plane, but we don't expose this through istioctl in a straightforward way. Users would need to run `istioctl version` + `istioctl ps` and compare the output. This often leads to hard to debug errors.

My sketch of this:

```
$ istioctl version
client version: 1.3.0-rc.3
control plane version: 1.3.0-rc.3
data plane version: 1.3.0 (50 proxies), 1.2.5 (12 proxies)
```

**Comment by esnible** [comment]
@howardjohn If the user does `istioctl version --short=false` or `-o json` I expect we want something like

```
sidecar version: version.BuildInfo{Version:"1.3.0", Pod: "productpage-1234-5678.default"}
```

We don't want to exec into each sidecar looking for UUIDs, or asking Kubernetes for image information do we?

**Comment by howardjohn** [comment]
@esnible definitely not exec into each proxy for performance reasons. I was thinking we could fetch it from pilot like `istioctl proxy-status` does. This may be a bit more complex than I originally thought, as we would need a new struct for sidecars since we don't know all the same information and need info like `Pod`.

I think we need *some* way to see this easily, but since we will have https://github.com/istio/istio/issues/17173 in 1.4 now I think this becomes lower priority. What do you think?

**Comment by esnible** [comment]
I have an implementation, not yet pushed because https://github.com/istio/pkg/pull/72 must merge first.  It does something reasonable when the output is `--short=false`, YAML, or JSON, and for the default it shows.

```
istioctl version
client version: 66676a6c52d4e00fdd24dfa35933fabc51965713-dirty
control plane version: 1.3.0
data plane version: 1.3.0: (9 proxies), 1.3.0-rc.2: (1 proxies)
```
